examples/all-clusters-app/esp32/main/CMakeLists.txt

```diff
@@ -49,7 +50,17 @@ idf_component_register(PRIV_INCLUDE_DIRS
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/door-lock-server"
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/ias-zone-server"
                       #${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/ias-zone-client
-                      PRIV_REQUIRES chip QRCode tft spidriver bt screen-framework)
+)
+if(${IDF_TARGET} STREQUAL "esp32")
+    idf_component_register(PRIV_INCLUDE_DIRS ${PRIV_INCLUDE_DIRS_GROUPS} 
+                           SRC_DIRS ${SRC_DIRS_GROUPS}
+                           PRIV_REQUIRES chip QRCode tft spidriver bt screen-framework)
+elseif(${IDF_TARGET} STREQUAL "esp32c3")
+    idf_component_register(PRIV_INCLUDE_DIRS ${PRIV_INCLUDE_DIRS_GROUPS}
+                                             "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/examples/common/screen-framework/include"
```

Comment:
Ideally for devkits without a display, this include path shouldn't be required at all. Please file an issue to clean up the display dependency.